src/app/pages/field-trip-summary/field-trip-summary.page.ts

```diff
@@ -104,13 +107,22 @@ export class FieldTripSummaryPage implements AfterViewInit {
 
 	getDeadSheep(): void {
 		const deadRegistrations = this.fieldTripInfo.registrations
-		.filter(reg => reg.registrationType === RegistrationType.Dead) as InjuredSheepRegistration[];
+		.filter(reg => reg.registrationType === RegistrationType.Dead) as DeadSheepRegistration[];
 
 		deadRegistrations.forEach((registration) => {
 			this.deadCount += registration.count;
 		});
 	}
 
+	getPredators(): void {
+		const predatorsRegistrations = this.fieldTripInfo.registrations
+		.filter(reg => reg.registrationType === RegistrationType.Predator) as PredatorRegistration[];
+
+		predatorsRegistrations.forEach(() => {
```

Comment:
Du kunne vel sikkert bare skrevet this.predators = predatorsRegistrations.length. Men dette fungerer jo også :))